src/calendar/day/multi-dot/index.js

```diff
@@ -43,13 +43,13 @@ class Day extends Component {
   }
 
   renderDots(marking) {
-    const baseDotStyle = [this.style.dot, this.style.visibleDot];
+    const baseDotStyle = [this.style.dot, this.style.visibleDot, marking.selected && this.style.selectedDot];
```

Comment:
`style.selectedDot` only hold backgroundColor from appStyle's `selectedDotColor`. That already treated in line 55. Why add it again here?